Index: kibana-5.1.1/bin/kibana
===================================================================
--- kibana-5.1.1.orig/bin/kibana
+++ kibana-5.1.1/bin/kibana
@@ -14,7 +14,7 @@ while [ -h "$SCRIPT" ] ; do
 done
 
 DIR="$(dirname "${SCRIPT}")/.."
-NODE="${DIR}/node/bin/node"
+NODE="/usr/bin/nodejs"
 test -x "$NODE" || NODE=$(which node)
 if [ ! -x "$NODE" ]; then
   echo "unable to find usable node.js executable."
